usaspending_api/common/custom_django_fields.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+
+
+class NumericField(models.Field):
```

Comment:
In Postgres, `numeric` is equivalent to `decimal`.  Could you not just use the built in `decimal` data type here or was the point of this specifically not to have to supply precision and scale values?

Reply:
My goal was the duplicate Broker's schema 100%. Their columns are defined as `numeric` without precision and scale. It isn't what I would choose, but that's the source data